autoscale/src/main/java/com/sequenceiq/periscope/controller/AutoScaleClusterCommonService.java

```diff
@@ -170,6 +171,12 @@ public String getResourceCrnByResourceName(String clusterName) {
         return getClusterByCrnOrName(NameOrCrn.ofName(clusterName)).getStackCrn();
     }
 
+    @Override
+    public String getEnvironmentCrnByResourceCrn(String resourceCrn) {
```

Comment:
Maybe this could be an optional return type on the interface method if this is not present all the time.

Reply:
as discussed, will do